src/base/install/getFilesInDir.m

```diff
@@ -0,0 +1,150 @@
+function files = getFilesInDir(varargin)
```

Comment:
How does this function work with a submodule? Which `.gitignore` file will be used?

Reply:
Depending on which option is used:
if any git dependent options (i.e.`untracked`, `tracked` or `ignored`) is used, it will use the result of whatever git uses (I assume the gitignore from the submodule). If 'ignoredByCOBRA' is used, it will use the COBRA git ignore, and if 'all' is used it will return all files.

Reply:
Would it make sense to print a message with the location of the `.gitignore` file if `printLevel > 0`?